StakeCraft/chains.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "../chains.schema.json",
+    "name": "StakeCraft",
+    "chains": [
+      {
+        "name": "agoric",
+        "address": "agoricvaloper1avjyymefy3wzepse09q3hrgnla3fjr72av43e9",
+        "restake": {
```

Comment:
Can you remove the restake section for all chains if you're not providing that service. Just needs name and address

Reply:
done!